proposals/0029-cooperative-vector.md

```diff
@@ -527,19 +589,24 @@ typedef enum D3D12_FEATURE {
     D3D12_FEATURE_COOPERATIVE_VECTOR;
 };
 
+// This is designed to match the ComponentType enum values but omits data 
+// types that are not currently specified to work with this API. The names are chosen
+// to more closely match those used by HLSL developers, as opposed to the ComponentType 
+// names that align with LLVM IR.
```

Comment:
@amarpMSFT - does lining the values up to match the DXIL constants with the runtime ones make sense?  I wonder if we really want to be tieing the runtime API with a particular interchange format at all?

Reply:
It doesn't hurt to line them up if the intended use is only with HLSL use anyhow.   If we ever ended up in a path where someone is feeding type choices of some kind from the API it would be handy if they lined up.